.github/workflows/tests.yml

```diff
@@ -57,7 +59,33 @@ jobs:
           ninja &&
           popd &&
           popd &&
-          cp bitwuzla/build/src/main/bitwuzla bin/ 
+          cp bitwuzla/build/src/main/bitwuzla bin/ &&
+          sudo apt-get install -y git cmake bison flex libboost-all-dev python2 perl &&
+          wget $STP_URL -nv -O stp.tar.gz &&
+          mkdir stp &&
+          tar xzf stp.tar.gz -C stp --strip-components=1 &&
+          pushd stp &&
+          ./scripts/deps/setup-gtest.sh &&
+          ./scripts/deps/setup-outputcheck.sh &&
+          ./scripts/deps/setup-cms.sh &&
+          ./scripts/deps/setup-minisat.sh &&
+          mkdir build &&
+          pushd build &&
+          cmake .. &&
+          cmake --build . &&
+          popd &&
+          popd &&
+          cp stp/build/stp bin/stp &&
+          sudo apt-get install -y gperf &&
+          wget $YICES2_URL -nv -O yices2.tar.gz &&
+          mkdir yices2 &&
+          tar xvf yices2.tar.gz -C yices2 --strip-components=1 &&
+          pushd yices2 &&
+          autoconf &&
+          ./configure --prefix=$PWD/out/ &&
+          make install &&
+          popd &&
+          cp yices2/out/bin/yices-smt2 bin/yices-smt2
```

Comment:
After compilation, does Yices produce an executable named `yices-smt2` or `yices_smt2`. This line seems to indicate that it produces `yices-smt2`, but the documentation says that it's `yices_smt2`.

Reply:
It's strange. I explain it in the documentation, but:
- after `make` it produces `yices_smt2`
- after `make install` it produces `yices-smt2`
quite frustrating. I documented this fact and also documented the fact that the executable that comes out of just `make` can simply be renamed. I think using `yices-smt2` is the correct option b/c this is the final name of the executable, and the name that's used when Yices is installed e.g. via `apt`.